wdl2cwl/main.py

```diff
@@ -396,7 +394,7 @@ def get_command(
     return new_command
 
 
-def get_output(expression: str, input_names: List[str]) -> str:
+def get_expression_output(expression: str, input_names: List[str]) -> str:
     """Get expression for outputs."""
```

Comment:
I was confused at first as I thought `get_output` was to `get_input`, but apparently it did something slightly different (I am not familiar with expressions for outputs as in the docs). I thought it would be OK to rename it and create a `get_output` that does something similar to `get_input`?